src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
Here's a compromise maybe: perhaps we could include some macros for parameter assertion, e.g. `ASSERT_CALLBACK(info[1])` would accomplish a few things:
 - first, and foremost, it would tell me that we assume the second parameter is a callback function
  - secondly it allows us to easily hide away the boilerplate code for doing a parameter check, and throwing with an appropriate warning about passed in type.
  - lastly, it solves my interest in self-documenting code

How does that sound?